packages/block-editor/src/components/block-list/index.js

```diff
@@ -222,7 +222,7 @@ class BlockList extends Component {
 						</AsyncModeProvider>
 					);
 				} ) }
-				<BlockListAppender rootClientId={ rootClientId } />
+				<BlockListAppender rootClientId={ rootClientId } disableDefaultInserter={ this.props.disableDefaultInserter } />
```

Comment:
Minor: could add `disableDefaultInserter` to the destructured props above this.

Reply:
Yes that's a good point. Outdated now so resolving.